util/environment/environment.go

```diff
@@ -1,33 +1,39 @@
 package environment
 
 import (
+	"os"
 	"sort"
 	"strings"
 )
 
-// MergeEnvironments merges env2 into env1, overwriting any existing variable
-// in env1, except for the "special" variables PATH, CPATH and LD_LIBRARY_PATH.
+var pathSeparator = string(os.PathListSeparator)
```

Comment:
I would call this variable `pathListSeparator`. The first thing that came to my mind when reading `pathSeparator` was `/` vs `\` as opposed to the intended `:` vs `;`.